### PR TITLE
fix Birthday

### PR DESCRIPTION
**Pull request by xhsien**
1. Birthday is optional.
2. Miscellaneous fix of documentation.

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Birthday.java [hunk 1]
```diff
@@ -24,21 +24,25 @@
...
-        this.value = birthday;
+        this.value = trimmedBirthday;
     }
 
     /**
-     * Returns true if a given string is a valid person email.
+     * Returns true if a given string is a valid date.
      */
-    static boolean isValidBirthday(String birthday) {
+    static boolean isValidBirthday(String test) {
```
Comment (chilipadiboy) [inline src/main/java/seedu/address/model/person/Birthday.java:37]: I feel that using test as a variable is misleading. Maybe use testBday or something instead?  Or just birthday

Comment (chilipadiboy) [reply src/main/java/seedu/address/model/person/Birthday.java:37]: Other than that it seems like your changes are exactly the same as mine except mine has some broken tests :(

**Review by chilipadiboy — APPROVED** [review]

File: src/main/java/seedu/address/model/person/Birthday.java (hunk 1 above)
Comment (xhsien) [reply src/main/java/seedu/address/model/person/Birthday.java:37]: Good point on `test` variable. I change it for the consistency throughout all `Email.java`, `Address.java` etc. Maybe we should update all of them.

Comment (Jemereny) [reply src/main/java/seedu/address/model/person/Birthday.java:37]: The test variable should be consistent. We should update it in the next PR or so.

**Comment by Jemereny** [comment]
Looks alright to merge.

**Pull request merged**